clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/arrays/TransferArraysV1Metadata.scala

```diff
@@ -0,0 +1,111 @@
+  hetHomvarRatio: Option[Double] = None,
+  hetPct: Option[Double] = None,
+  idatDirName: Option[String] = None,
+  imagingDate: Option[OffsetDateTime] = None,
+  isLatest: Option[Boolean] = None,
+  isZcalled: Option[Boolean] = None,
+  modifiedAt: Option[OffsetDateTime] = None,
+  notes: Option[String] = None,
+  novelSnps: Option[Long] = None,
+  numAutocallCalls: Option[Long] = None,
+  numCalls: Option[Long] = None,
+  numInDbSnp: Option[Long] = None,
+  numNoCalls: Option[Long] = None,
+  numSingletons: Option[Long] = None,
+  p95Green: Option[Int] = None,
+  p95Red: Option[Int] = None,
+  paramsFile: Option[URI] = None,
+  pctDbsnp: Option[Double] = None,
+  pipelineVersion: Option[Symbol] = None,
+  refDict: Option[URI] = None,
+  refFasta: Option[URI] = None,
+  refFastaIndex: Option[URI] = None,
+  reportedGender: Option[Symbol] = None,
+  researchProjectId: Option[String] = None,
+  sampleAlias: Option[String] = None,
+  scannerName: Option[Symbol] = None,
+  totalAssays: Option[Long] = None,
+  totalIndels: Option[Long] = None,
+  totalSnps: Option[Long] = None,
+  workflowEndDate: Option[OffsetDateTime] = None,
+  workflowStartDate: Option[OffsetDateTime] = None,
+  zcallThresholdsFile: Option[URI] = None,
+  zcallVersion: Option[Symbol] = None
+) extends TransferMetadata[TransferArraysV1Metadata] {
+
+  override def pathsToDelete: Seq[URI] =
+    Seq.concat(
+      beadPoolManifestFile,
+      clusterFile,
+      dbSnpVcf,
+      dbSnpVcfIndex,
+      extendedChipManifestFile,
+      fileOfIdatFilenames,
+      genderClusterFile,
+      haplotypeDatabaseFile,
+      paramsFile,
+      refDict,
+      refFasta,
+      refFastaIndex,
+      zcallThresholdsFile
+    )
+
+  override def mapMove(
+    pathMapper: (Option[URI], String) => Option[URI]
+  ): TransferArraysV1Metadata = this.copy(
+    beadPoolManifestFile = pathMapper(beadPoolManifestFile, ArraysExtensions.BpmExtension),
+    clusterFile = pathMapper(clusterFile, ArraysExtensions.EgtExtension),
+    dbSnpVcf = pathMapper(dbSnpVcf, ArraysExtensions.VcfGzExtension),
+    dbSnpVcfIndex = pathMapper(dbSnpVcfIndex, ArraysExtensions.VcfGzTbiExtension),
+    extendedChipManifestFile =
+      pathMapper(extendedChipManifestFile, ArraysExtensions.CsvExtension),
+    fileOfIdatFilenames =
+      pathMapper(fileOfIdatFilenames, ArraysExtensions.IdatsTxtExtension),
+    genderClusterFile = pathMapper(genderClusterFile, ArraysExtensions.EgtExtension),
+    haplotypeDatabaseFile =
+      pathMapper(haplotypeDatabaseFile, ArraysExtensions.TxtExtension),
+    paramsFile = pathMapper(paramsFile, ArraysExtensions.TxtExtension),
+    refDict = pathMapper(refDict, ArraysExtensions.DictExtension),
+    refFasta = pathMapper(refFasta, ArraysExtensions.FastaExtension),
+    refFastaIndex = pathMapper(refFastaIndex, ArraysExtensions.FastaFaiExtension),
+    zcallThresholdsFile =
+      pathMapper(zcallThresholdsFile, ArraysExtensions.EgtThresholdsTxtExtension)
```

Comment:
An extra twist: this `copy` is used to build the set of moves that should happen for data-delivery. Not sure if there's anything we should leave out, like we do with cram metrics.

Reply:
@ktibbett probably also knows the list of things that should be delivered and what shouldn't move around. From working on the delivery side of things, it looks like we'll want to deliver:
1. vcf
2. idat(s)
3. gtc

And any indexes for those things (I'm guessing).